examples/multimodal_vision/internvl3_readme.md

```diff
@@ -0,0 +1,183 @@
+# Quantizing InternVL3-8B-hf
+This file shows the example of quantizing InternVL3-8B-hf. If you want to quantize InternVL3 model, you can convert internvl-chat format to internvl-hf format with the script [internvl_custom2hf.py](https://github.com/OpenGVLab/InternVL/blob/main/internvl_chat/tools/internvl_custom2hf.py).
```

Comment:
To be clear, this conversion is not necessary for the above example? Is `OpenGVLab/InternVL3-8B-hf` already in the correct format?